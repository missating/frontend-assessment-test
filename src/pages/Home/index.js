import React from 'react';


const Home = () => (
  <div className="app-container">
    Home page
  </div>
)

export default Home;
