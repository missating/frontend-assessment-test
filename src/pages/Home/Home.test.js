// react library
import React from 'react';
import { shallow } from 'enzyme';

// oages
import Home from './index'

describe('HotelList Component', () => {
  it('should be rendered properly', () => {
    // eslint-disable-next-line react/jsx-props-no-spreading
    const wrapper = shallow(<Home />);
    expect(wrapper.find('div').length).toEqual(1);
    expect(wrapper).toMatchSnapshot();
  });
})
