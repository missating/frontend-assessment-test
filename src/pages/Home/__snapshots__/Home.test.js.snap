// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HotelList Component should be rendered properly 1`] = `
<div
  className="app-container"
>
  <HotelList />
</div>
`;
